guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -158,3 +158,19 @@ endif::[]
 
 . Optional: If you made manual edits to DNS or DHCP configuration files, check and restore any changes required to the DNS and DHCP configuration files using the backups made earlier.
 . Optional: If you use custom repositories, ensure that you enable these custom repositories after the upgrade completes.
+
+.Upgrading {SmartProxyServers} using remote execution
+
+. Create a backup or take a snapshot.
++
+For more information on backups, see {AdministeringDocURL}backing-up-satellite-server-and-capsule-server[Backing Up {ProjectServer} and {SmartProxyServer}] in _{AdministeringDocTitle}_.
```

Comment:
This link looks wrong to me. I had a look at it on the "3.3" branch: 

````
$ rg -i "backing-up-"
guides/common/modules/con_backing-up-server-and-proxy.adoc
1:[id="backing-up-{project-context}-server-and-{smart-proxy-context}_{context}"]

$ rg "upgrading_capsule_server.adoc"
guides/doc-Upgrading_and_Updating/master.adoc
63:include::topics/upgrading_capsule_server.adoc[leveloffset=+2]
````

It looks like that this file is included for all build targets which is confirmed by the list of changed HTML files as posted by the GH bot. This means that the link should work for all build targets. > please use attributes.

Reply:
Sorry for the delay, I have replaced the filename with the correct one.